roles/default/kiali-deploy/templates/openshift/clusterrole.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
can you rename this so it is more specific - in [a similar way we have specific names for some role templates](https://github.com/kiali/kiali-operator/tree/master/roles/default/kiali-deploy/templates/openshift) this is for supporting oauth so name this `clusterrole-oauth.yaml`.

Reply:
I kept it intentionally generic in case there were other cluster scoped objects that we wanted to manage in the future similar to how `role.yaml` and `rolebinding.yaml` are named but I can rename if that's not a concern.

Reply:
If we're renaming the files should the clusterrole/binding themselves also be renamed to `<instance>-<namespace>-oauth`?

Reply:
Not a concern about the file names - these are even openshift specific so hopefully we won't need to provide more openshift-specific perms. But if we do, we'll worry about it later.

Might not be a bad idea to rename the resources themselves too - makes it clear what their purpose is for.